tooling/src/hypothesistooling/__main__.py

```diff
@@ -63,6 +63,18 @@ def check_installed():
     don't fail to run if a previous install failed midway)."""
 
 
+@task()
```

Comment:
I don't think it's worth having this as an independent task; we only make it a function in order to call it from both `lint()` and `format()`.

Reply:
Done! 👍